third_party/move/move-compiler-v2/tests/more-v1/parser/spec_parsing_ok.exp

```diff
@@ -48,7 +48,7 @@ error: expression construct not supported in specifications
 69 │         ensures RET = {let y = x; y + 1};
    │                 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-error: cannot pass `|(num, num)|num` to a function which expects argument of type `|(num, num)|bool`
+error: cannot pass `|(num, num)|num with copy+drop+store` to a function which expects argument of type `|(num, num)|bool`
```

Comment:
I wonder if we could remove abilities (with copy+drop+store) from the type error message if the error is not due to abilities mismatch.